app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -38,8 +38,39 @@
       <td class="hidden-tablet hidden-xs"><%= host.primary_interface.try(:subnet) %></td>
       <td class="hidden-tablet hidden-xs"><%= disc_report_column(host) %></td>
       <td>
+        <!-- Modal -->
```

Comment:
Yeah not big deal, its just I have 100 hosts per page setting and page is quite big.